### PR TITLE
Cc 16351 b2c product measurement units

### PR DESCRIPTION
**Pull request by Mb-Hala**

### CODE REVIEW

File: tests/api/b2c/glue/weight_unit_endpoints/product_measurement_units/negative.robot [hunk 1]
```diff
@@ -0,0 +1,18 @@
+*** Settings ***
+Suite Setup       SuiteSetup
+Resource    ../../../../../../resources/common/common_api.robot
+
+*** Test Cases ***
```
Comment (OKrivtsova) [inline tests/api/b2c/glue/weight_unit_endpoints/product_measurement_units/negative.robot:5]: please add

*** Settings ***

Test Setup        TestSetup
Default Tags    glue

*** Test Cases ***
ENABLER
    TestSetup

NOTE: also run your tests before creating a PR to make they they pass, without these lines ALL tests fail. You should not submit on-working tests for PR

File: tests/api/b2c/glue/weight_unit_endpoints/product_measurement_units/positive.robot [hunk 2]
```diff
@@ -0,0 +1,17 @@
+*** Settings ***
+Suite Setup       SuiteSetup
+Resource    ../../../../../../resources/common/common_api.robot
+
+*** Test Cases ***
```
Comment (OKrivtsova) [inline tests/api/b2c/glue/weight_unit_endpoints/product_measurement_units/positive.robot:5]: please add

*** Settings ***

Test Setup        TestSetup
Default Tags    glue

*** Test Cases ***
ENABLER
    TestSetup

NOTE: also run your tests before creating a PR to make they they pass, without these lines ALL tests fail. You should not submit on-working tests for PR

File: tests/api/b2c/glue/weight_unit_endpoints/sales_units/negative.robot [hunk 3]
```diff
@@ -0,0 +1,25 @@
+*** Settings ***
+Suite Setup       SuiteSetup
+Resource    ../../../../../../resources/common/common_api.robot
+
+*** Test Cases ***
```
Comment (OKrivtsova) [inline tests/api/b2c/glue/weight_unit_endpoints/sales_units/negative.robot:5]: please add
*** Settings ***

Test Setup        TestSetup
Default Tags    glue

*** Test Cases ***
ENABLER
    TestSetup

NOTE: also run your tests before creating a PR to make they they pass, without these lines ALL tests fail. You should not submit on-working tests for PR

File: tests/api/b2c/glue/weight_unit_endpoints/sales_units/positive.robot [hunk 4]
```diff
@@ -0,0 +1,43 @@
+*** Settings ***
+Suite Setup       SuiteSetup
+Resource    ../../../../../../resources/common/common_api.robot
+
+*** Test Cases ***
```
Comment (OKrivtsova) [inline tests/api/b2c/glue/weight_unit_endpoints/sales_units/positive.robot:5]: please add

*** Settings ***

Test Setup        TestSetup
Default Tags    glue

*** Test Cases ***
ENABLER
    TestSetup

NOTE: also run your tests before creating a PR to make they they pass, without these lines ALL tests fail. You should not submit on-working tests for PR

File: tests/api/b2c/glue/weight_unit_endpoints/sales_units/positive.robot [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+Get_sales_units_for_product_without_sales_units
```
Comment (OKrivtsova) [inline tests/api/b2c/glue/weight_unit_endpoints/sales_units/positive.robot:7]: even after all the setting added tests fail on https://glue.de.b2c.internal-testing.demo-spryker.com with environmewt api_b2c - please fix it
image

**Comment by vadym-uskov** [comment]
This endpoint is not needed for b2c and mp_b2c shops

**Pull request closed without merging**